ParkingFinder/services/parking_space.py

```diff
@@ -0,0 +1,57 @@
+from sqlalchemy.orm.exc import NoResultFound
+from schematics.exceptions import ValidationError
+from tornado.gen import coroutine, Return
+
+from ParkingFinder.base.errors import (
+    NotFound,
+    InvalidEntity,
+)
+from ParkingFinder.repositories.real_time_repository import RealTimeRepository
+
+from clay import config
+
+logger = config.get_logger('parkingspace')
+
+
+class ParkingSpaceService(object):
+
```

Comment:
Add methods and handler endpoint if needed to stop realtime location service . Once the realtime is established, there is no method allows front end to stop and remove from db. Please consider if requesting_user cancel the realtime and what to return when the waiting_user send the request and vice versa.